cjworkbench/i18n/trans.py

```diff
@@ -38,7 +46,24 @@ def trans(message_id, *, default, context=None, parameters={}):
     For code parsing reasons, respect the following order when passing keyword arguments:
         `message_id` and then `default` and then `context` and then everything else
     """
-    return _get_translations(get_language()).trans(
+    return do_trans(
+        get_language(),
+        message_id,
+        default=default,
+        context=context,
+        parameters=parameters,
+    )
+
+
+def do_trans(locale_id, message_id, *, default, context=None, parameters={}):
```

Comment:
Indeed, I added it during experimenting with module serialization last month and I thought it will be useful in the future. However, it is not certain and it will not be hard to add it again if needed.